presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -67,6 +70,9 @@ class SignalHandler;
 class TaskManager;
 class TaskResource;
 class PeriodicTaskManager;
+#ifdef PRESTO_ENABLE_LINUX_MEMORY_CHECKER
+class LinuxMemoryChecker;
```

Comment:
If you are including the header on line 26 you don't need the forward declaration here for the member.
I suggest moving the header include to the cpp and keep the forward declaration.